nsqadmin/static/js/models/topic.js

```diff
@@ -10,6 +12,16 @@ var Topic = Backbone.Model.extend({
 
     url: function() {
         return AppState.url('/topics/' + encodeURIComponent(this.get('name')));
+    },
+
+    parse: function(response) {
+        response['nodes'] = _.map(response['nodes'] || [], function(node) {
+            var nodeParts = node['node'].split(':');
+            node['show_broadcast_address'] = node['hostname'] !== nodeParts[0];
```

Comment:
Noticing from your screenshot, can you make the comparison of hostname case insensitive here and the other function?